app/frontend/styles/_grid.scss

```diff
@@ -3,3 +3,9 @@
     text-align: right;
   }
 }
+
+.app-grid-column--grey {
+  background-color: #ebf2f6;
```

Comment:
@DFE-Digital/find-and-apply-designers Is this a random colour? I couldn't find it in the list of predefined variables.

Reply:
> @DFE-Digital/find-and-apply-designers Is this a random colour? I couldn't find it in the list of predefined variables.

It's from DfE design manual and was suggested by our head of design.